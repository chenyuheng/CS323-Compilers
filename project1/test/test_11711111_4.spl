int a;
int b;
int d;
char c;
int;
float;
char;

int main(char arg1[32], char arg2, int arg3[3][4][5][66666]) {
    int i = 0xfff;
    char o = '\xt0';
    int t = 0X5Aax;
    int 0ok = 0;
    (i) = 0xaaa
    return really_main();
}

int lalala[3][4][5][666];
float hahaha;
char nanana[32];

int really_main() {
    int return_value = main(nanana, c, lalala);
    return return_value;
}
