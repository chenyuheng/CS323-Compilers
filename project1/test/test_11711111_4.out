Error type A at Line 11: unknown lexeme '\xt0'
Error type A at Line 12: unknown lexeme 0X5Aax
Error type A at Line 13: unknown lexeme 0ok
Error type B at Line 14: Missing semicolon ';'
